README.md

```diff
@@ -568,7 +568,10 @@ fsdp_config:
   fsdp_transformer_layer_cls_to_wrap: LlamaDecoderLayer
 ```
 
-- llama Deepspeed: append `ACCELERATE_USE_DEEPSPEED=true` in front of finetune command
+- llama Deepspeed: append `ACCELERATE_USE_DEEPSPEED=true` in front of finetune command and set the following:
```

Comment:
```suggestion
- llama Deepspeed: prepend `ACCELERATE_USE_DEEPSPEED=true` in front of finetune command and set the following:
```

Reply:
Resolved due to setting this automatically